### ISSUE TITLE
WCAG 2.0 or Section 508

### ISSUE THREAD

**Issue by accessiblewebuk** [18F/accessibility#156]
https://pages.18f.gov/accessibility/ says "Here at 18F we follow WCAG2.0 AA as our standard for accessibility." https://pages.18f.gov/accessibility/checklist/ says "For more detail on accessibilty standards, please see WCAG2.0 AA"

Some clarity is needed to avoid misunderstanding. Is section 508 in line with WCAG 2.0 now? If not, then these pages aren't really clear on which standard they are referring to.

**Comment by carodew** [comment]
Hi @accessiblewebuk – Section 508 currently is not inline with WCAG 2.0 but the refresh is based on that standard. At 18F we follow WCAG 2.0 AA as it's more thorough, not because it's required by law. For a good comparison of the two standards, see https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/background/comparison-table-of-wcag2-to-existing-508-standards

Does that answer your question?

**Comment by accessiblewebuk** [comment]
Thanks @carodew - that makes sense. I think I assumed it would be 508 or a modified version of 508. Looking at this for GDS use as we do use WCAG 2.0 AA and that should become part of UK law for the public sector within the next two years.
